ResourceStatusSystem/Agent/EmailAgent.py

```diff
@@ -38,55 +41,25 @@ def initialize( self ):
 
   def execute( self ):
 
-    #if the file exists and it is not empty
-    if os.path.isfile(self.cacheFile) and os.stat(self.cacheFile).st_size:
+    conn = sqlite3.connect(self.cacheFile)
```

Comment:
I suggest to use the `with` statement to interact with sqlite3: the code will be more compact and safe: `with sqlite3.connect(self.cacheFile) as conn` and then you don't need to invoke `conn.close()`